cmd/kube-apiserver/app/options/options.go

```diff
@@ -48,6 +48,7 @@ type ServerRunOptions struct {
 	Authorization           *kubeoptions.BuiltInAuthorizationOptions
 	CloudProvider           *kubeoptions.CloudProviderOptions
 	APIEnablement           *genericoptions.APIEnablementOptions
+	NetworkProxy            *genericoptions.NetworkProxyOptions
```

Comment:
I'm super sorry I couldn't get to this earlier-- unfortunately with the rename I think now these other references to "NetworkProxy" will be super confusing :/

Reply:
+1. Fixed.